frontends/mit-open/src/types/settings.d.ts

```diff
@@ -2,7 +2,7 @@
 export type User = {
   id: number | null
   is_authenticated: boolean
-  is_public_list_editor: boolean
+  is_article_editor: boolean
```

Comment:
`is_public_list_editor` was removed; we just forgot to remove it here.

`is_article_editor` already existed on `window.SETTINGS.user`, we just hadn't used it yet.